Singer.API/Controllers/CareRegistrationController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Singer.Models;
+using Singer.Services.Interfaces;
+
+namespace Singer.Controllers
+{
+   [Authorize]
+   [Route("api/careregistration")]
+   public class CareRegistrationController : Controller
+   {
+      private readonly IEventRegistrationService _eventRegistrationService;
+
+      public CareRegistrationController(IEventRegistrationService eventRegistrationService)
+      {
+         _eventRegistrationService = eventRegistrationService;
+      }
+
+      [HttpPost]
+      public async Task<ActionResult<CareRegistrationResultDTO>> Create(
+         [FromBody] CreateCareRegistrationDTO createCareRegistration)
+      {
+         var result = await _eventRegistrationService.Create(createCareRegistration.EventRegistrationType
+            , createCareRegistration.CareUserIds, createCareRegistration.StartDateTime.LocalDateTime,
+            createCareRegistration.EndDateTime.LocalDateTime);
+         return new CareRegistrationResultDTO()
```

Comment:
Shouldn't this be returning a `Created(nameof(Get), ...` instead of a plain DTO?

Reply:
@BerendWouters ? You resolved the comment without pushing a change or commenting

Reply:
Ah, yes.
Will fix.